modules/virt-binding-devices-vfio-driver.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices-vfio-driver_{context}"]
+= Binding a PCI device to the VFIO driver
+
+To bind a PCI device to the VFIO driver, find the values for `vendor-ID` and `device-ID` and add them to the `/etc/modprobe.d/vfio.conf` file. Then, configure the `MachineConfig` object file for the VFIO driver and load the new driver.
+
+.Prerequisites
+. The Intel or AMD hardware must be supported by the CPU and enabled in the BIOS settings.
+
+.Procedure
+. To bind a device to the VFIO driver, apply the kernel module to the `MachineConfig` object file. The following example shows the `MachineConfig` object file.
+
++
+.Example
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig <1>
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <2>
+  name: 100-worker-vfiopci-configuration <3>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,b3B0aW9ucyB2ZmlvLXBjaSBpZHM9MTBkZToxZWI4Cg== <4>
+        mode: 420 <5>
+        overwrite: true <6>
+        path: /etc/modprobe.d/vfio.conf <7>
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,dmZpby1wY2kK <8>
+        mode: 420
+        overwrite: true
+        path: /etc/modules-load.d/vfio-pci.conf <9>
+
+----
+<1> The type of object being configured is `MachineConfig`.
+<2> Applies the new kernel argument only to worker nodes.
+<3> The number (100) indicates the rank for this `MachineConfig` among all `MachineConfig` objects. The name given to this `MachineConfig` indicates the purpose of this object.
+<4> The encoded text for `source` (`b3B0aW9ucyB2ZmlvLXBjaSBpZHM9MTBkZToxZWI4Cg`) translates to `options vfio-pci ids=10de:1eb8`. The `vendor-ID` value (10de) and the `device-ID` value (1eb8) applies to a single device that binds to the VFIO driver. You can add a list of multiple devices for `source` with their vendor and device information.
```

Comment:
the `==` should be included in `b3B0aW9ucyB2ZmlvLXBjaSBpZHM9MTBkZToxZWI4Cg`

Perhaps we should mention how to generate this:
```
echo “options vfio-pci ids=10de:1eb8” | base64
```

Reply:
Updated the content.